geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/pubsub/SubscriptionsIntegrationTest.java

```diff
@@ -53,49 +54,53 @@
     };
   }
 
+  final Subscriptions subscriptions = new Subscriptions();
+
+  private int dummyCount;
```

Comment:
Would it be worth resetting this value in a `@Before` method?

Reply:
It gets reset for each test method because a new instance of the test class is created.
But given this test is doing concurrency testing I am changing this counter to an AtomicInteger to make sure two concurrent threads do not get the same dummyCount value.